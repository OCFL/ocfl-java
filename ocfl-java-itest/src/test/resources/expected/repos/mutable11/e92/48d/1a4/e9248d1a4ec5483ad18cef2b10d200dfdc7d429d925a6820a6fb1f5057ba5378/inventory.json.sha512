4f64704b3f59181a59a2b27654769bac427a6671a76416776c2dc4e9bd8f5e8f0801af4b11f3286368f8c41b07d26be853d1c2108a56f0c502cb644301f9c656  inventory.json
