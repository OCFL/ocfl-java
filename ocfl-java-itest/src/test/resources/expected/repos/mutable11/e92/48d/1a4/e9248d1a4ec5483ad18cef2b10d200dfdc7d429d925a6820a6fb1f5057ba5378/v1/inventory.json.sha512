9bddb6164b1db49c3ac758dd3be82647cd3f1b39adbe5b350560209ac4ffefcde7534e31c6b71d355b94cca17698a934a8d52f054a3994d46c2cc92ddfd75ee4  inventory.json
