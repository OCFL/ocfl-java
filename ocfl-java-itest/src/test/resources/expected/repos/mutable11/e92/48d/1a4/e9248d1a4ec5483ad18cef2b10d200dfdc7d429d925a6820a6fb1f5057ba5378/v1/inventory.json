{
  "id" : "object_1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v1",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : { },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "Auto-generated empty object version.",
      "user" : {
        "name" : "ocfl-java",
        "address" : "https://github.com/OCFL/ocfl-java"
      },
      "state" : { }
    }
  }
}